pages/training.php

```diff
@@ -142,9 +144,13 @@ function training() {
 			// Check for sensei training boost
 			if($player->sensei_id != 0) {
 				$sensei_boost = SenseiManager::getStudentBoostBySensei($player->sensei_id, $system);
-				if ($train_type == $sensei_boost['specialization'] . "_skill" || $train_type == $sensei_boost['specialization']) {
+                if ($train_type == $sensei_boost['specialization'] . "_skill" || $train_type == $sensei_boost['specialization']) {
```

Comment:
Maybe we could simplify this to a `SenseiManager::getTrainingBoostForTrainType($player->sensei_id, $train_type, $system)`

which in most cases returns 0 or null, but in these two cases returns `$sensei_boost['boost_primary']`

Reply:
Modified in commit.

Reply:
Changed in latest commit.